EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3084,55 +3084,55 @@ QUEST_JOBSTEP_20150730_003083	The Squire Master says he needs the Seal of the Le
+QUEST_JOBSTEP_20150803_003117	You've collected the keepsakes, of which have a heavy aura about them. Return to the Necromancer Master.
 QUEST_JOBSTEP_20150803_003118	Meet Vaidotas at Siauliai Miners' Village
 QUEST_JOBSTEP_20150803_003119	Vaidotas told you that in order to investigate Crystal Mine, he needs the Orb Recorder of Past Memories. Obtain the Orb Recorder of Past Memories from the Chronomancer Master.
 QUEST_JOBSTEP_20150803_003120	You've borrowed the orb that records the memories of the past. Return to Vaidotas.
 QUEST_JOBSTEP_20150803_003121	You've obtained the orb that records the memories of the past. Talk with Vaidotas.
-QUEST_JOBSTEP_20150803_003122	Vaidotas wants you to investigate the incident that occurred in Crystal Mine. With the glass orb that reflects the past which the Chronomancer Master gave to you, accomplish the exploration rate 100% in all regions of the 1st and 2nd district of Crystal Mine.
+QUEST_JOBSTEP_20150803_003122	Vaidotas wants you to investigate the incident that occurred in Crystal Mine. With the glass orb that the Chronomancer Master gave to you, and its ability to reflect the past, gain a 100% exploration rate in the 1st and 2nd district of Crystal Mine.
 QUEST_JOBSTEP_20150803_003123	The memories of Crystal Mine are recorded in the orb. Return to Vaidotas.
-QUEST_JOBSTEP_20150803_003124	Give the present secretly to the Scout Master
-QUEST_JOBSTEP_20150803_003125	The Rogue Master wants to give a surprise present to the Scout Master who once was very close to him. Find the Scout Master in Gateway of Great King and put the present secretly in his pocket.
-QUEST_JOBSTEP_20150803_003126	You've succeeded putting the present secretly without being detected. Return to the Rogue Master.
+QUEST_JOBSTEP_20150803_003124	Secretly give the present to the Scout Master
+QUEST_JOBSTEP_20150803_003125	The Rogue Master wants to give a surprise present to the Scout Master. Find the Scout Master in Gateway of Great King and place the present secretly in her pocket.  It seems that this is fairly important for him - they must have been close at one point.
+QUEST_JOBSTEP_20150803_003126	You've succeeded in placing the present without being detected. Return to the Rogue Master.
 QUEST_JOBSTEP_20150803_003127	Search for the traces of Lydia Schaffen
-QUEST_JOBSTEP_20150803_003128	The Fletcher Master wants to look for the bow which the first the Fletcher Master made for Lydia Schaffen. Find the bow of the first Flectcher Master from 31_1 of Canyon Area.
+QUEST_JOBSTEP_20150803_003128	The Fletcher Master wants you to look for the bow that the first Fletcher Master made for Lydia Schaffen. Find the bow from 31_1 of Canyon Area.
 QUEST_JOBSTEP_20150803_003129	Give them to the Fletcher Master
-QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the Fletcher Master.
+QUEST_JOBSTEP_20150803_003130	You've found the bow of the first Fletcher Master. Return to the Fletcher Master that exists today.
```

Comment:
I was gonna go with current originally, but I put exists today to hint towards a larger gap of time.  I'll do current though.